fastmri/data/subsample.py

```diff
@@ -53,16 +53,78 @@ def __init__(self, center_fractions: Sequence[float], accelerations: Sequence[in
 
```

Comment:
Would be a good idea to document here how the base `MaskFunc` controls the high-level logic of mask creation and what subclasses should re-implement for specific functionality. As I understand, subclasses should for the most part just overwrite `calculate_acceleration_mask()`?